plugin-locationlayer/src/main/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayerPlugin.java

```diff
@@ -525,10 +525,19 @@ private void updateLocation(final Location location) {
     }
 
     staleStateManager.updateLatestLocationTime();
+
     if (lastLocation != null) {
-      locationLayerAnimator.feedNewLocation(lastLocation, location);
+      LatLng previousLatLngTarget = new LatLng(lastLocation);
+      LatLng latLngTarget = new LatLng(location);
+      locationLayerAnimator.feedNewLatLng(previousLatLngTarget, latLngTarget);
```

Comment:
Shouldn't `previousLatLngTarget` be taken from the current camera position, not from the last location?